### PR TITLE
Added document section with required Links

### LINKED ISSUE #71: docs: Add a documentation Section in the README.md
### Is your feature request related to a problem?

Add a section in README.md which directs us to the wiki of the project.

### Describe the solution you'd like

Create a "Documentation" section with:
- link to the wiki for further information regarding the community
- link to open sessions which include the meeting minutes, 
- link to the video of GSoC: https://youtu.be/3A746GppZ0Y
- link of adding a video of setting up the project: https://youtu.be/_b2RQGbYN9w
- link of the question in zulip chats.
 
You can check these resources on this [zulip message](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs/topic/Open.20Sessions/near/225236028)

### PR DESCRIPTION
**Pull request by atharvashirdhankar**
### Description

Added document section with required Links

Fixes #71

### Type of Change:

- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?

N/A

### Checklist:

- [ ] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged
- [ ] I have written Kotlin Docs whenever is applicable

### CODE REVIEW

**Comment by atharvashirdhankar** [comment]
This PR should solve issue no. #71

**Comment by Aaishpra** [comment]
Can you add the issue number in the PR description after `Fixes # ` so that the PR closes the issue

**Comment by atharvashirdhankar** [comment]
Ok I will do it as soon as possible

**Comment by atharvashirdhankar** [comment]
Done

**Comment by codesankalp** [comment]
@StarTrooper08 Add `Fixes #<issue_number>` in PR description and not in the PR title.
If you read the description you will see Fixes just add the issue number there.

**Comment by atharvashirdhankar** [comment]
Done ✅

File: README.md
Comment (Aaishpra) [inline README.md]:
```suggestion
1. **anitab-org wiki** [link](https://github.com/anitab-org/anitab-forms-web/wiki)
```
Do the same with other links also

Comment (atharvashirdhankar) [reply README.md]: Ok

**Comment by atharvashirdhankar** [comment]
@Aaishpra do the changes I have made now is right??

File: README.md
Comment (codesankalp) [inline README.md]: @Aaishpra Is an open session link necessary in this? Because this may change in future.

File: README.md
Comment (codesankalp) [inline README.md]:
```suggestion
6. [Announcement in Zulip chats](https://anitab-org.zulipchat.com/#narrow/stream/213491-announcements): In this stream, all the important decision are announced by admins related to anitab-org opensource.
```
What do you think @Aaishpra about this representation i.e. link followed by some detail? This can be done for all the links.

**Review by codesankalp — CHANGES_REQUESTED** [review]

File: README.md
Comment (atharvashirdhankar) [reply README.md]: Can I do this changes?

Comment (Aaishpra) [reply README.md]: This is a good idea but I am still not sure about adding announcement and questions stream links in the readme as they are common streams and not specifically dedicated to this project. Just adding links to our project stream should be fine. 🤔 What do you think?

File: README.md
Comment (Aaishpra) [reply README.md]: Yes, it can definitely change, Shall we add the document link of our open sessions instead of this?

File: README.md
Comment (Aaishpra) [reply README.md]: Although we can definitely add details for other streams.
@StarTrooper08 can you add these details by yourself or would you like some help?

Comment (atharvashirdhankar) [reply README.md]: > This is a good idea but I am still not sure about adding announcement and questions stream links in the readme as they are common streams and not specifically dedicated to this project. Just adding links to our project stream should be fine. 🤔 What do you think?

I have one idea ,
we can just add one zulip chat link instead of those 2 channels links of zulip chat.


What do you say??

Comment (Aaishpra) [reply README.md]: Ya,  and the link should take you to the anitab-forms stream.(although I think we already have one in the badge).
Let's see what @codesankalp says

Comment (atharvashirdhankar) [reply README.md]: > Although we can definitely add details for other streams.
> @StarTrooper08 can you add these details by yourself or would you like some help? 

Ok I will first try of my own and if stuck somewhere I will take help .

Comment (atharvashirdhankar) [reply README.md]: > Ya,  and the link should take you to the anitab-forms stream.(although I think we already have one in the badge).
> Let's see what @codesankalp says

Ok

**Review by codesankalp — DISMISSED** [review]

**Comment by StarTrooper08** [comment]
image

@codesankalp I have done the mentioned changes should I make a PR.

File: README.md
Comment (vj-codes) [inline README.md]:
```suggestion
1. [anitab-forms wiki](https://github.com/anitab-org/anitab-forms-web/wiki) : Wiki page  
```

**Comment by Aaishpra** [comment]
@StarTrooper08 any updates?

**Comment by atharvashirdhankar** [comment]
I m sorry, for the delay.

**Review by Aaishpra — DISMISSED** [review]
Great work

**Comment by Aaishpra** [comment]
@codesankalp @isabelcosta have a look here

File: README.md
Comment (codesankalp) [inline README.md]:
```suggestion
1. [anitab-forms wiki](https://github.com/anitab-org/anitab-forms-web/wiki): Wiki page  
2. [Open Sessions](https://meet.google.com/eqb-nuut-kqm): Link to join weekly meetings for anitab-forms.
3. [GSoC youtube video link](https://youtu.be/3A746GppZ0Y): Final video demo of anitab-forms developed during GSoC 2020
4. [Setting up the project youtube video link](https://youtu.be/_b2RQGbYN9w): Here is the youtube video link for setting up the project for development purposes.
5. [Question in Zulip chats link](https://anitab-org.zulipchat.com/#narrow/stream/223070-questions): In this zulip channel, all the Q&A happens related to anitab-org opensource.
6. [Announcement in Zulip chats](https://anitab-org.zulipchat.com/#narrow/stream/213491-announcements): In this stream, all the important decisions are announced by admins related to anitab-org opensource.

```

**Review by codesankalp — CHANGES_REQUESTED** [review]

**Review by isabelcosta — DISMISSED** [review]
Thank you for contributing @StarTrooper08 🙌🏾

**Comment by isabelcosta** [comment]
@StarTrooper08 Can you run prettier, so the lint check passes please?

**Comment by atharvashirdhankar** [comment]
Ok @isabelcosta I'll do it.
But how can I do it 😅.
Can I get help!

**Comment by isabelcosta** [comment]
@StarTrooper08 of course you can get help :) I just have to ping other maintainers, because I am not very well versed with frontend 😅 @Aaishpra @codesankalp 
I took a look at the [readme](https://github.com/anitab-org/anitab-forms-web#to-run-eslint) and it does say you can run: `npm run lint:fix`. Can you do this and push the changes?

**Review by Aaishpra — COMMENTED** [review]
@StarTrooper08  Isabel told you the correct command or if you use yarn use `yarn lint-fix` once the tests are passing I can approve the PR,

**Comment by codesankalp** [comment]
@StarTrooper08 Rebasing your branch with the current developed branch will also make the lining test pass.

**Comment by atharvashirdhankar** [comment]
Ok

**Comment by isabelcosta** [comment]
@StarTrooper08 are you still working on this? If not, let us know so we can make the ticket available again :)

**Comment by atharvashirdhankar** [comment]
Yes yes I was stucked in some other work.

**Comment by atharvashirdhankar** [comment]
I used the command `npm run lint:fix`.
But still, it's showing 1 failing check.

**Comment by codesankalp** [comment]
> I used the command `npm run lint:fix`. But still, it's showing 1 failing check.

Have you rebased the branch?
After rebasing try to run the command and then push the rebased branch.

**Comment by atharvashirdhankar** [comment]
@isabelcosta @codesankalp please have a look

**Comment by codesankalp** [comment]
@StarTrooper08 Can you please resolve the merge conflicts?

**Comment by codesankalp** [comment]
> @StarTrooper08 Can you please resolve the merge conflicts?

To do this replace the develop branch package-lock.json file with your local file.
Also, I see you haven't rebased the branch (try this):
```
git remote add upstream git@github.com:anitab-org/anitab-forms-web.git 
git fetch upstream
git rebase upstream/develop
npm install
npm run lint:fix
git push
```

**Pull request closed without merging**
